server/fleet/activities.go

```diff
@@ -1448,6 +1450,52 @@ func (a ActivityTypeInstalledSoftware) Documentation() (activity, details, detai
 }`
 }
 
+type ActivityTypeAddedSoftware struct {
+	SoftwareTitle   string `json:"software_title"`
+	SoftwarePackage string `json:"software_package"`
+	TeamName        string `json:"team_name"`
+}
+
+func (a ActivityTypeAddedSoftware) ActivityName() string {
+	return "added_software"
+}
+
+func (a ActivityTypeAddedSoftware) Documentation() (string, string, string) {
+	return `Generated when a software installer is uploaded to Fleet.`, `This activity contains the following fields:
+- "software_title": Name of the software.
+- "software_package": Filename of the installer.
+- "team_name": Name of the team to which this software was added. Empty string if it was added to no team.`,
+		`{
+  "software_title": "Falcon.app",
+  "software_package": "FalconSensor-6.44.pkg",
+  "team_name": "foo"
```

Comment:
I think we generally use `Workstations` as example team name, to be more realistic.

Reply:
Lol I did that at first, but then decided to search in the file for an example of a team name and
the first result for some reason was one on l384 that used "foo".